modules/monitoring-exposing-custom-application-metrics-for-horizontal-pod-autoscaling.adoc

```diff
@@ -165,7 +170,7 @@ metadata:
 spec:
   service:
     name: prometheus-adapter
-    namespace: default
+    namespace: custom-prometheus
```

Comment:
Line 166
```apiVersion: apiregistration.k8s.io/v1beta1```
if we use oc 46 client, it would throw out warning
```W0924 04:14:00.794618   10444 warnings.go:67] apiregistration.k8s.io/v1beta1 APIService is deprecated in v1.19+, unavailable in v1.22+; use apiregistration.k8s.io/v1 APIService```
it is better change to ```apiregistration.k8s.io/v1```

Reply:
@juzhao thank you for highlighting this. I have now amended the line on my local clone and will push the change to my fork shortly.